frontend/src/pages/modelServing/screens/projects/ModelServingPlatform.tsx

```diff
@@ -197,6 +218,11 @@ const ModelServingPlatform: React.FC = () => {
                       <GalleryItem>
                         <EmptyMultiModelServingCard />
                       </GalleryItem>
+                      {isNIMModelServingAvailable && isNIMAPIKeyValid && (
```

Comment:
FWIW, you should have this down to `isNIMAvailable` -- avoid unnecessarily unravelled logic whenever possible. Neither of these values have unique purpose on their own.